plugins/com.google.cloud.tools.eclipse.ui.util.test/src/com/google/cloud/tools/eclipse/ui/util/databinding/ProjectSelectorValidatorTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.eclipse.ui.util.databinding;
+
+import static org.hamcrest.CoreMatchers.is;
```

Comment:
This test file is actually a renamed file which had this import at the beginning, but I think I'll remove this hamcrest to avoid this deprecation mark.